src/commands/message.js

```diff
@@ -191,6 +191,8 @@ class CommandMessage {
 					const err = new CommandFormatError(this);
 					return this.reply(err.message);
 				}
+				
+				this.command.onCancelled(this);
```

Comment:
The base `Command` class does not have an `onCancelled` method, this would not work on any command that doesn't implement `onCancelled`, such as the internal commands.